<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="1" file="subnet_calculator_tests.py" name="subnet_calculator_tests.TestConvertNetmaskBitsToBinary-20201009092748" skipped="0" tests="7" time="0.012" timestamp="2020-10-09T09:27:48">
	<testcase classname="subnet_calculator_tests.TestConvertNetmaskBitsToBinary" name="test_maximum_ipv4_netmask" time="0.001" timestamp="2020-10-09T09:27:48"/>
	<testcase classname="subnet_calculator_tests.TestConvertNetmaskBitsToBinary" name="test_minimum_netmask" time="0.001" timestamp="2020-10-09T09:27:48"/>
	<testcase classname="subnet_calculator_tests.TestConvertNetmaskBitsToBinary" name="test_netmask_one_above_minimum" time="0.001" timestamp="2020-10-09T09:27:48"/>
	<testcase classname="subnet_calculator_tests.TestConvertNetmaskBitsToBinary" name="test_netmask_well_above_minimum" time="0.001" timestamp="2020-10-09T09:27:48"/>
	<testcase classname="subnet_calculator_tests.TestConvertNetmaskBitsToBinary" name="test_one_below_maximum_ipv4_netmask" time="0.001" timestamp="2020-10-09T09:27:48"/>
	<testcase classname="subnet_calculator_tests.TestConvertNetmaskBitsToBinary" name="test_well_below_maximum_ipv4_netmask" time="0.001" timestamp="2020-10-09T09:27:48"/>
	<testcase classname="subnet_calculator_tests.TestConvertNetmaskBitsToBinary" name="test_maximum_ipv6_netmask" time="0.005" timestamp="2020-10-09T09:27:48">
		<failure message="'11111111.11111111.11111111.11111111' != '.11111111.11111111.11111111.11111111.1111[90 chars]1111'
- 11111111.11111111.11111111.11111111
+ .11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111
" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/usr/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.6/unittest/case.py", line 605, in run
    testMethod()
  File "/home/student/Documents/cs3280project2/src/unittest/python/subnet_calculator_tests.py", line 655, in test_maximum_ipv6_netmask
    self.assertEquals('11111111.11111111.11111111.11111111', binary_netmask)
  File "/usr/lib/python3.6/unittest/case.py", line 1321, in deprecated_func
    return original_func(*args, **kwargs)
  File "/usr/lib/python3.6/unittest/case.py", line 829, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib/python3.6/unittest/case.py", line 1203, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib/python3.6/unittest/case.py", line 670, in fail
    raise self.failureException(msg)
AssertionError: '11111111.11111111.11111111.11111111' != '.11111111.11111111.11111111.11111111.1111[90 chars]1111'
- 11111111.11111111.11111111.11111111
+ .11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111.11111111

]]>		</failure>
	</testcase>
</testsuite>
